Polar/README.md

```diff
@@ -1,3 +1,73 @@
 # Polar Datacube Generation
 
-Information dedicated to the generation of the Polar Datacube.
\ No newline at end of file
+Information dedicated to the generation of the Polar Datacube.
+
+## Cube generation process
+
+The cube generation process is divided in four phases:
+
+### 1. Downloading required raw datasets
+
+Datasets were downloaded via the source information provided in `cube.geojson`. Datasets were downloaded in different formats: `.nc`, `.tif`, `.shp`. Some datasets were compressed and this step includes the extraction of the data.
+
+To download the datasets it is required to run the following file:
+
+```
+# Download all files
+inputs-collect/download-sources.py
+```
+
+### 2. Preprocessing datasets
+
+The raw data is preprocessed in order to create a single input `.zarr` cube per dataset. The preprocessing steps might involve time resampling and/or spatial resampling according to the dataset. The preprocessing code for each dataset is found at the `inputs-preprocess` folder. Note that some datasets have multiple parts according to the preprocessing steps that were applied:
+
+```
+# Curie Depth Estimates
+inputs-preprocess/curie-depth.py # Convert files to zarr
+inputs-preprocess/curie-depth-resample.py # Spatial resampling
+
+# Geothermal Heat Flow
+inputs-preprocess/geothermal-heat-flow.py # Convert files to zarr
+inputs-preprocess/geothermal-heat-flow-resample.py # Spatial resampling
+
+# Geothermal Heat Flow Uncertainty
+inputs-preprocess/geothermal-heat-flow-uncertainty.py # Convert files to zarr
+inputs-preprocess/geothermal-heat-flow-uncertainty-resample.py # Spatial resampling
+
+# Magnetic Anomaly
+inputs-preprocess/magnetic-anomaly.py # Convert files to zarr
+inputs-preprocess/magnetic-anomaly-resample.py # Spatial resampling
+
+# Subglacial Lakes
+inputs-preprocess/subglacial-lakes-2013-2017.py # Convert files to zarr
+inputs-preprocess/subglacial-lakes-2013-2017-resample.py # Spatial resampling
+
+# Ice Thickness
+inputs-preprocess/bedrock-ice-thickness.py # Convert files to zarr
+inputs-preprocess/bedrock-ice-thickness-resample.py # Spatial resampling
+```
+
+After preprocessing, all `.zarr` cubes were uploaded to `s3://deep-esdl-input/` by running the following file:
```

Comment:
Please add a note, that this bucket location needs to be adjusted by any user not belonging to the DeepESDL cube-gen team to point to a bucket provided by the user.